Resources/Prototypes/Entities/Clothing/Head/hats.yml

```diff
@@ -101,7 +101,7 @@
     - WhitelistChameleon
 
 - type: entity
-  parent: ClothingHeadBase
+  parent: [ClothingHeadBase, BaseRestrictedContraband]
```

Comment:
```suggestion
  parent: [ClothingHeadBase, BaseCommandContraband]
```

Reply:
I meant to put it into engineering contraband, thanks for the catch.